### ISSUE TITLE
Problem with logging into docker.

### ISSUE THREAD

**Issue by PKulesza-iswireless** [goharbor/harbor#17110]
Hi,
I'm a new Harbor user and I am just beginning to understand how it works. I have successfully installed Harbor 2.5.2 and am now trying to push helm chart using docker. Unfortunately, I am not able to log into docker. 

`docker login https://mydomain.test.com`
`Username: admin`
`Password: Harbor12345`
`Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/auth": dial unix /var/run/docker.sock: connect: permission denied`
As you can see, I used the default password that is used to login to Harbor. Could someone tell me if docker have different credentials  than Harbor, or if I set something up wrong.

Thanks in advance for any advice.

**Comment by MinerYang** [comment]
Hi @PKulesza-iswireless ,

Welcome and Thx for interested in Harbor.
you might need to handle the docker permission problem first , pls refer to this doc https://docs.docker.com/engine/install/linux-postinstall/ .
Then you might need to down load the registry certificate and put it into the .docker/certs/<harbor-hostname> directory 
or any other places that this certificate  can be trusted.

Best,
Miner

**Comment by PKulesza-iswireless** [comment]
Hi @MinerYang 

I was able to get rid of this permission error but now I'm getting the following error.
`Error response from daemon: Get "https://mydomain.test.com/v2/": x509: cannot validate certificate for mydomain.test.com because it doesn't contain any IP SANs
`
I don't know if this could be a problem but I generated my certificate on a linux machine and I don't use a domain name. 
I generated the certificates for my machine's public IP.

Best regards,
Piotr Kulesza
